systems/plants/RigidBodyManipulator.cpp

```diff
@@ -248,7 +248,7 @@ void getFiniteIndexes(T const & v, std::vector<int> &finite_indexes)
 {
   finite_indexes.clear();
   const size_t n = v.size();
-  for (size_t x = 0; x < n; x++)
+  for (int x = 0; x < n; x++)
```

Comment:
Just realized this is my code.  It probably should have been a vector of size_t, looking back at it.